csharp/Ice/session/server/SessionFactory.cs

```diff
@@ -1,13 +1,15 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 
 using System;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
`using System.Threading.Tasks;` is not required

```suggestion
using System.Threading;
```